modules/ROOT/pages/deployment/general/general-info.adoc

```diff
@@ -280,6 +282,28 @@ NOTE: You must set this environment variable to a valid path when using the manu
 
 WARNING: The location must exclusively be used by Infinite Scale. Writing into this location not using Infinite Scale is strictly discouraged to avoid any unexpected behaviour. 
 
+== Initialize Infinite Scale
+
+To prepare Infinite Scale to have a base for further configuration and for recurring starts, you have to initalize it first. To do so, run:
+
+[source,bash]
+----
+ocis init
+----
+
+You can add command line parameters. To see which one are available type:
+
+[source,bash]
+----
+ocis init --help
+----
+
+Command line parameters are benificial if you e.g. want to handover all necessary parameters without getting to any questionaire or when you want to define the admin password by yourself not getting a random one.
+
+IMPORTANT: The command line option `--force-overwrite` is only intended for developer usage. If you set this option, your config will be overwritten, your data, if any is present will persist, but it will not be accessible anymore. This is, among other things, because the issuer (short _iss_, part of openID Connect) will be overwritten.
```

Comment:
```suggestion
IMPORTANT: The command line option `--force-overwrite` is only intended for developer usage. If you set this option, your config will be overwritten, your data, if any is present, will persist, but it will not be accessible anymore. This is, among other things, because the issuer (short _iss_ part of openID Connect) will be overwritten.
```